code/modules/antagonists/ert/ert.dm

```diff
@@ -224,6 +224,11 @@
 	missiondesc += "<BR> <B>3.</B> If Security is found to be violating the rights of citizens, detain them as per your authority as Spinward Stellar Coalition officers."
 	missiondesc += "<BR> <B>4.</B> If the situation demands it, evacuate the station. Otherwise, remain on station and keep the peace."
 	to_chat(owner,missiondesc)
+	var/policy = get_policy(ROLE_FAMILIES)
+	if(policy)
+		to_chat(owner, policy)
```

Comment:
are you sure that the police should be getting the same policy messages as gangsters?

Reply:
was this intentionally resolved? It looks like nothing changed and there's no comment here to suggest it's intended.

Reply:
Intentional, my bad. Forgot to actually hit the comment button. Due to how gamemodes work we only have one ROLE_X for families.